packages/scoutgame/src/claims/claimDailyRewardAction.ts

```diff
@@ -12,8 +12,7 @@ export const claimDailyRewardAction = authActionClient
     const data = await claimDailyReward({
```

Comment:
we should add validation now, since we have a deterministic timezone. if dayOfWeek is not today, it should throw an error

Reply:
Updated!